### PR TITLE
Fix "Get Started" at the top being 404

### PR DESCRIPTION
**Pull request by akihironitta**
## What does this PR do?
Following up #12163.

## Before submitting

- [n/a] Was this **discussed/approved** via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- [n/a] Did you make sure to **update the documentation** with your changes? (if necessary)
- [n/a] Did you write any **new necessary tests**? (not for typos and docs)
- [x] Did you verify new and **existing tests pass** locally with your changes?
- [n/a] Did you list all the **breaking changes** introduced by this pull request?
- [n/a] Did you **update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)**? (not for typos, docs, test updates, or internal minor changes/refactorings)

## PR review

Anyone in the community is welcome to review the PR.
Before you start reviewing make sure you have read [Review guidelines](https://github.com/PyTorchLightning/pytorch-lightning/wiki/Review-guidelines). In short, see the following bullet-list:

- [x] Is this pull request ready for review? (if not, please submit in draft mode)
- [x] Check that all items from **Before submitting** are resolved
- [x] Make sure the title is self-explanatory and the description concisely explains the PR
- [x] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?

Make sure you had fun coding 🙃

cc @borda @rohitgr7

### CODE REVIEW

**Review by kaushikb11 — APPROVED** [review]

**Review by ananthsub — APPROVED** [review]

**Review by ethanwharris — APPROVED** [review]

**Review by rohitgr7 — APPROVED** [review]
thank you!!

File: docs/source/starter/converting.rst [hunk 1]
```diff
@@ -193,9 +193,9 @@ and all the :class:`~torch.nn.Module` instances initialized inside ``LightningMo
             super().__init__()
             self.register_buffer("running_mean", torch.zeros(num_features))
 
-If you still need to access the current device, you can use ``self.device`` anywhere in ``LightningModule`` except ``__init__`` method. You are initializing a
-:class:`~torch.Tensor` within ``LightningModule.__init__`` method and want it to be moved to the device automatically you must :meth:`~torch.nn.Module.register_buffer`
-to register it as a parameter.
+If you still need to access the current device, you can use ``self.device`` anywhere in ``LightningModule`` except ``__init__`` and ``setup`` methods.
+You are initializing a :class:`~torch.Tensor` within ``LightningModule.__init__`` method and want it to be moved to the device automatically you must
+:meth:`~torch.nn.Module.register_buffer` to register it as a parameter.
```
Comment (awaelchli) [inline docs/source/starter/converting.rst:198]: Error 404, grammar not found

**Review by awaelchli — APPROVED** [review]

File: docs/source/starter/converting.rst (hunk 1 above)
Comment (awaelchli) [reply docs/source/starter/converting.rst:198]: #12243
